src/Tribe/Main.php

```diff
@@ -80,7 +80,7 @@ class Tribe__Events__Main {
 		public $postOrganizerTypeArgs = array();
 
 		public static $tribeUrl = 'http://tri.be/';
-		public static $tecUrl = 'http://theeventscalendar.com/';
+		public static $tecUrl = 'https://theeventscalendar.com/';
```

Comment:
Just a thought (doesn't stop us from merging this change though) - but would an [m.tri.be](http://m.tri.be) link afford more insights/tracking potential?